### PR TITLE
[2.0.4] dmctl: support read master addr from env

### PR DESCRIPTION
**Pull request by Ehco1996**
### What is changed, added, or deleted? (Required)

update dmctl doc to note that `master_addr` can be read from env var.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/dm/pull/1726

### Do your changes match any of the following descriptions?

- [x] Change aliases

### CODE REVIEW

File: zh/dmctl-introduction.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ Use "dmctl [command] --help" for more information about a command.
 >
 > + 一条 dmctl 命令只能跟一个任务操作
 > + 任务操作只能放在 dmctl 命令的最后
+> + 从 v2.0.4 版本开始, `-master-addr`参数支持从环境变量(DM_MASTER_ADDR)里读取
```
Comment (CharLotteiu) [inline zh/dmctl-introduction.md:79]:
```suggestion
> + 从 v2.0.4 版本开始，支持从环境变量(DM_MASTER_ADDR)里读取 `-master-addr` 参数
```
1. 2.0.4 预计什么时候发版？用户现在使用哪个版本的 DM 能感受到这个功能？还是建议文档和发版同步，不要脱节。
2. 具体读取的操作和方式目前没有介绍，也没有提供可以参考的介绍，是否预期读者已经熟悉这个操作？

Comment (GMHDBJD) [reply zh/dmctl-introduction.md:79]: 这个会在2.0.4 release后才合进master。
最好下面加个 export DM_MASTER_ADDR 的例子

Comment (Ehco1996) [reply zh/dmctl-introduction.md:79]: 好的，那等发了再来和这个 pr。

**Comment by Ehco1996** [comment]
@CharLotteiu  @GMHDBJD 
DM v2.0.4 has been released and this PR is ready to be merged.

**Review by CharLotteiu — APPROVED** [review]

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: en/dmctl-introduction.md [hunk 2]
```diff
@@ -75,13 +75,17 @@ The command mode differs from the interactive mode in that you need to append th
 > **Note:**
 >
 > + A dmctl command must be followed by only one task operation.
+> + Starting from v2.0.4, the `-master-addr` parameter supports reading from environment variables (DM_MASTER_ADDR).
```
Comment (CharLotteiu) [inline en/dmctl-introduction.md:78]:
```suggestion
> + Starting from v2.0.4, DM supports reading the `-master-addr` parameter from the environment variable `DM_MASTER_ADDR`.
```

File: zh/dmctl-introduction.md [hunk 3]
```diff
@@ -75,13 +75,17 @@ Use "dmctl [command] --help" for more information about a command.
 > **注意：**
 >
 > + 一条 dmctl 命令只能跟一个任务操作
+> + 从 v2.0.4 版本开始，支持从环境变量(DM_MASTER_ADDR)里读取 `-master-addr` 参数
```
Comment (CharLotteiu) [inline zh/dmctl-introduction.md:78]:
```suggestion
> + 从 v2.0.4 版本开始，支持从环境变量 (DM_MASTER_ADDR) 里读取 `-master-addr` 参数
```

**Pull request merged**